docs/acsengine.md

```diff
@@ -139,11 +140,16 @@ Build acs-engine:
   2. Type `cmd` to open a command prompt
   3. Type `mkdir %GOPATH%` to create your gopath
   4. Type `cd %GOPATH%`
-  5. Type `go get github.com/Azure/acs-engine` to download acs-engine from GitHub
+  5. Type `go get -d github.com/Azure/acs-engine` to download acs-engine from GitHub
   6. Type `go get all` to get the supporting components
-  7. Type `cd %GOPATH%\src\github.com\Azure\acs-engine`
-  8. Type `go build` to build the project
-  9. Run `bin\acs-engine` to see the command line parameters
+  7. Type `go get github.com/jteeuwen/go-bindata/...`
```

Comment:
Should use "go get -u github.com/jteeuwen/go-bindata/...", else credential is needed.